### ISSUE TITLE
[Bug] Wrong storage info used space

### ISSUE THREAD

**Issue by Dibort** [MauriceNino/dashdot#277]
### Description of the bug

Storage info shows wrong 1.6% used on my internal hard drive

![image](https://user-images.githubusercontent.com/7732931/184563177-a9866fcd-55af-4c94-994e-0fd9d90de477.png)

_I'd like to just see storage info of /dev/sda2 if possible._

### Relevant log output

```shell
$ df -h
Filesystem      Size  Used Avail Use% Mounted on
tmpfs           389M  1,7M  388M   1% /run
/dev/sda3        51G   15G   34G  30% /
tmpfs           1,9G     0  1,9G   0% /dev/shm
tmpfs           5,0M     0  5,0M   0% /run/lock
/dev/sda2       866G  291G  531G  36% /data
tmpfs           389M  4,0K  389M   1% /run/user/1000


$ docker exec dash df -h
Filesystem                Size      Used Available Use% Mounted on
overlay                  50.4G     14.3G     33.5G  30% /
tmpfs                    64.0M         0     64.0M   0% /dev
shm                      64.0M         0     64.0M   0% /dev/shm
/dev/sda3                50.4G     14.3G     33.5G  30% /mnt/host
udev                      1.8G         0      1.8G   0% /mnt/host/dev
tmpfs                     1.9G         0      1.9G   0% /mnt/host/dev/shm
tmpfs                   388.9M      1.7M    387.3M   0% /mnt/host/run
tmpfs                     5.0M         0      5.0M   0% /mnt/host/run/lock
tmpfs                   388.9M      1.7M    387.3M   0% /mnt/host/run/snapd/ns
tmpfs                   388.9M      4.0K    388.9M   0% /mnt/host/run/user/1000
/dev/loop4               55.6M     55.6M         0 100% /mnt/host/snap/core18/2409
/dev/loop0               80.0M     80.0M         0 100% /mnt/host/snap/lxd/22911
/dev/loop2               62.0M     62.0M         0 100% /mnt/host/snap/core20/1587
/dev/loop5               80.0M     80.0M         0 100% /mnt/host/snap/lxd/22923
/dev/loop3               55.6M     55.6M         0 100% /mnt/host/snap/core18/2538
/dev/loop7               47.0M     47.0M         0 100% /mnt/host/snap/snapd/16292
/dev/loop6               43.8M     43.8M         0 100% /mnt/host/snap/snapd/15177
/dev/sda2               865.1G    298.5G    522.6G  36% /mnt/host/data
overlay                  50.4G     14.3G     33.5G  30% /mnt/host/var/lib/docker/overlay2/0f440aa797491d157bf4608b868bb916811263acb2f3d0d0d02b2393e313135f/merged
overlay                  50.4G     14.3G     33.5G  30% /mnt/host/var/lib/docker/overlay2/86da54392a3573be49b900b2a59433bfce2fca8e610fdb78d47bab874d6c88ed/merged
overlay                  50.4G     14.3G     33.5G  30% /mnt/host/var/lib/docker/overlay2/91824a84da67c9c140905fb6139f7b4cc0ef4fcb749df611d4a4e9d609ae692e/merged
/dev/loop8               62.0M     62.0M         0 100% /mnt/host/snap/core20/1593
overlay                  50.4G     14.3G     33.5G  30% /mnt/host/var/lib/docker/overlay2/5b4ff19dbcd53fe5cb28c5006bfb2e3224d8f2251e8f9cf5bc137d94aabf5ba4/merged
overlay                  50.4G     14.3G     33.5G  30% /mnt/host/var/lib/docker/overlay2/5b4ff19dbcd53fe5cb28c5006bfb2e3224d8f2251e8f9cf5bc137d94aabf5ba4/merged
tmpfs                    64.0M         0     64.0M   0% /mnt/host/var/lib/docker/overlay2/5b4ff19dbcd53fe5cb28c5006bfb2e3224d8f2251e8f9cf5bc137d94aabf5ba4/merged/dev
shm                      64.0M         0     64.0M   0% /mnt/host/var/lib/docker/overlay2/5b4ff19dbcd53fe5cb28c5006bfb2e3224d8f2251e8f9cf5bc137d94aabf5ba4/merged/dev/shm
/dev/sda3                50.4G     14.3G     33.5G  30% /etc/resolv.conf
/dev/sda3                50.4G     14.3G     33.5G  30% /etc/hostname
/dev/sda3                50.4G     14.3G     33.5G  30% /etc/hosts

$ docker exec dash yarn cli raw-data --storage
yarn run v1.22.19
Disk Layout: [
  {
    device: '/dev/sda',
    type: 'HD',
    name: 'ST1000LM024 HN-M',
    vendor: 'Seagate',
    size: 1000204886016,
    bytesPerSector: null,
    totalCylinders: null,
    totalHeads: null,
    totalSectors: null,
    totalTracks: null,
    tracksPerCylinder: null,
    sectorsPerTrack: null,
    firmwareRevision: '0001',
    serialNum: '',
    interfaceType: 'SATA',
    smartStatus: 'unknown',
    temperature: null
  }
]
FS Size: [
  {
    fs: 'overlay',
    type: 'overlay',
    size: 54116986880,
    used: 15365545984,
    available: 35969257472,
    use: 29.93,
    mount: '/'
  },
  {
    fs: '/dev/sda3',
    type: 'ext4',
    size: 54116986880,
    used: 15365545984,
    available: 35969257472,
    use: 29.93,
    mount: '/mnt/host'
  },
  {
    fs: '/dev/loop4',
    type: 'squashfs',
    size: 58327040,
    used: 58327040,
    available: 0,
    use: 100,
    mount: '/mnt/host/snap/core18/2409'
  },
  {
    fs: '/dev/loop0',
    type: 'squashfs',
    size: 83886080,
    used: 83886080,
    available: 0,
    use: 100,
    mount: '/mnt/host/snap/lxd/22911'
  },
  {
    fs: '/dev/loop2',
    type: 'squashfs',
    size: 65011712,
    used: 65011712,
    available: 0,
    use: 100,
    mount: '/mnt/host/snap/core20/1587'
  },
  {
    fs: '/dev/loop5',
    type: 'squashfs',
    size: 83886080,
    used: 83886080,
    available: 0,
    use: 100,
    mount: '/mnt/host/snap/lxd/22923'
  },
  {
    fs: '/dev/loop3',
    type: 'squashfs',
    size: 58327040,
    used: 58327040,
    available: 0,
    use: 100,
    mount: '/mnt/host/snap/core18/2538'
  },
  {
    fs: '/dev/loop7',
    type: 'squashfs',
    size: 49283072,
    used: 49283072,
    available: 0,
    use: 100,
    mount: '/mnt/host/snap/snapd/16292'
  },
  {
    fs: '/dev/loop6',
    type: 'squashfs',
    size: 45875200,
    used: 45875200,
    available: 0,
    use: 100,
    mount: '/mnt/host/snap/snapd/15177'
  },
  {
    fs: '/dev/sda2',
    type: 'ext4',
    size: 928909111296,
    used: 320652161024,
    available: 560995532800,
    use: 36.37,
    mount: '/mnt/host/data'
  },
  {
    fs: '/dev/loop8',
    type: 'squashfs',
    size: 65011712,
    used: 65011712,
    available: 0,
    use: 100,
    mount: '/mnt/host/snap/core20/1593'
  }
]
Block Devices: [
  {
    name: 'sda',
    type: 'disk',
    fsType: '',
    mount: '',
    size: 1000204886016,
    physical: 'HDD',
    uuid: '',
    label: '',
    model: 'ST1000LM024 HN-M',
    serial: '',
    removable: false,
    protocol: 'sata',
    group: undefined
  },
  {
    name: 'loop0',
    type: 'loop',
    fsType: 'squashfs',
    mount: '/mnt/host/snap/lxd/22911',
    size: 83832832,
    physical: '',
    uuid: '',
    label: '',
    model: '',
    serial: '',
    removable: false,
    protocol: '',
    group: undefined
  },
  {
    name: 'loop2',
    type: 'loop',
    fsType: 'squashfs',
    mount: '/mnt/host/snap/core20/1587',
    size: 64970752,
    physical: '',
    uuid: '',
    label: '',
    model: '',
    serial: '',
    removable: false,
    protocol: '',
    group: undefined
  },
  {
    name: 'loop3',
    type: 'loop',
    fsType: 'squashfs',
    mount: '/mnt/host/snap/core18/2538',
    size: 58261504,
    physical: '',
    uuid: '',
    label: '',
    model: '',
    serial: '',
    removable: false,
    protocol: '',
    group: undefined
  },
  {
    name: 'loop4',
    type: 'loop',
    fsType: 'squashfs',
    mount: '/mnt/host/snap/core18/2409',
    size: 58232832,
    physical: '',
    uuid: '',
    label: '',
    model: '',
    serial: '',
    removable: false,
    protocol: '',
    group: undefined
  },
  {
    name: 'loop5',
    type: 'loop',
    fsType: 'squashfs',
    mount: '/mnt/host/snap/lxd/22923',
    size: 83832832,
    physical: '',
    uuid: '',
    label: '',
    model: '',
    serial: '',
    removable: false,
    protocol: '',
    group: undefined
  },
  {
    name: 'loop6',
    type: 'loop',
    fsType: 'squashfs',
    mount: '/mnt/host/snap/snapd/15177',
    size: 45748224,
    physical: '',
    uuid: '',
    label: '',
    model: '',
    serial: '',
    removable: false,
    protocol: '',
    group: undefined
  },
  {
    name: 'loop7',
    type: 'loop',
    fsType: 'squashfs',
    mount: '/mnt/host/snap/snapd/16292',
    size: 49242112,
    physical: '',
    uuid: '',
    label: '',
    model: '',
    serial: '',
    removable: false,
    protocol: '',
    group: undefined
  },
  {
    name: 'loop8',
    type: 'loop',
    fsType: 'squashfs',
    mount: '/mnt/host/snap/core20/1593',
    size: 64970752,
    physical: '',
    uuid: '',
    label: '',
    model: '',
    serial: '',
    removable: false,
    protocol: '',
    group: undefined
  },
  {
    name: 'sda1',
    type: 'part',
    fsType: '',
    mount: '',
    size: 1048576,
    physical: '',
    uuid: '',
    label: '',
    model: '',
    serial: '',
    removable: false,
    protocol: '',
    group: undefined
  },
  {
    name: 'sda2',
    type: 'part',
    fsType: 'ext4',
    mount: '/mnt/host/data',
    size: 944892805120,
    physical: '',
    uuid: '65940997-d0b3-4eaa-93d2-5cb53b895bb1',
    label: '',
    model: '',
    serial: '',
    removable: false,
    protocol: '',
    group: undefined
  },
  {
    name: 'sda3',
    type: 'part',
    fsType: 'ext4',
    mount: '/mnt/host',
    size: 55308189696,
    physical: '',
    uuid: 'd832f0e5-2e31-4e1d-aa49-38d880d6c37a',
    label: '',
    model: '',
    serial: '',
    removable: false,
    protocol: '',
    group: undefined
  },
  {
    name: 'sr0',
    type: 'rom',
    fsType: '',
    mount: '',
    size: 1073741312,
    physical: 'CD/DVD',
    uuid: '',
    label: '',
    model: 'CDDVDW TS-L633C ',
    serial: '',
    removable: true,
    protocol: 'sata',
    group: undefined
  }
]
```

### Info output of dashdot cli

```shell
yarn run v1.22.19
$ node dist/apps/cli/main.js info
INFO
=========
Yarn: 1.22.19
Node: v18.7.0
Dash: 4.3.9

Cwd: /app
Hash: e238d7147db6cea8c73358988325ce8b4ce5d2af
In Docker: true
In Podman: false
In Docker (env): true
Done in 1.36s.
```

### What browsers are you seeing the problem on?

Chrome

### Where is your instance running?

Linux Server

**Comment by MauriceNino** [comment]
Thanks for reporting this issue! It has been resolved in the next release.

**Comment by MauriceNino** [comment]
:tada: This issue has been resolved in version 4.3.10

Please check the [changelog](https://github.com/MauriceNino/dashdot/blob/main/.github/CHANGELOG.md) for more details.
